syntax = "proto3";
import "google/protobuf/empty.proto";
//python -m grpc_tools.protoc -I./ --python_out=. --grpc_python_out=. ./server.proto
//python -m grpc_tools.protoc --mypy_out=. --proto_path=. server.proto

message ProcessedMeteoData {
  int32 lb_id = 1;
  float air_wellness=2;
  int32 timestamp = 3;
}

message ProcessedPollutionData {
  float pollution_coefficient = 1;
  int32 lb_id = 2;
  int32 timestamp = 4;
}

service Server {
  rpc ReceivedMeteoData (ProcessedMeteoData) returns (google.protobuf.Empty) {}
  rpc ReceivedPollutionData (ProcessedPollutionData) returns (google.protobuf.Empty) {}
}

